examples/ethercat/process_data_objects.py

```diff
@@ -64,32 +63,14 @@ def create_pdo_maps(self) -> Tuple[PDOMap]:
             register = self.servo.dictionary.registers(1)[rpdo_register]
             self.rpdo_map.add_registers(register)
 
-    def check_state(self, state: SlaveState) -> None:
-        """Check if the slave reached the requested state.
-
-        Args:
-            state: Requested state.
-
-        Raises:
-            ILError: If state is not reached.
-
-        """
-        if self.master.state_check(state.value, 50_000) != state.value:
-            self.master.read_state()
-            if self.slave.state != state.value:
-                raise ILError(
-                    f"{self.slave.name} did not reach {SlaveState(state).name}."
-                    f" {hex(self.slave.al_status)} {pysoem.al_status_code_to_string(self.slave.al_status)}"
-                )
-
     def process_data_loop(self) -> None:
         """Process inputs and generate outputs."""
         while not self._pd_thread_stop_event.is_set():
             self.servo.process_pdo_inputs()
             for item in self.tpdo_map.items:
                 TPDO_REGISTERS[item.register.identifier] = item.value
             for item in self.rpdo_map.items:
-                RPDO_REGISTERS[item.register.identifier] += 100
+                RPDO_REGISTERS["CL_POS_SET_POINT_VALUE"] += 100
```

Comment:
You are adding 100 two times, right? Shouldn't this for loop be removed?

Reply:
The for loop is right, but the adding line should be outside of the loop

Reply:
88889db